src/types/app-factory.ts

```diff
@@ -170,44 +179,95 @@ export class AppFactory {
     this._paramsMethods = this.getParamsMethods()
   }
 
-  /** Get parameters to define create and deploy related calls to the current app */
+  /** Get parameters to create transactions (create and deploy related calls) for the current app.
+   *
+   * A good mental model for this is that these parameters represent a deferred transaction creation.
+   * @example Create a transaction in the future using Algorand Client
+   * ```typescript
+   * const createAppParams = appFactory.params.create({method: 'create_method', args: [123, 'hello']})
+   * // ...
+   * await algorand.send.AppCreateMethodCall(createAppParams)
+   * ```
+   * @example Define a nested transaction as an ABI argument
+   * ```typescript
+   * const createAppParams = appFactory.params.create({method: 'create_method', args: [123, 'hello']})
+   * await appClient.send.call({method: 'my_method', args: [createAppParams]})
+   * ```
+   */
   get params() {
     return this._paramsMethods
   }
 
-  /**
-   * Creates an instance of the app and returns the result of the creation
-   * transaction and an app client to interact with that app instance.
-   *
-   * Performs deploy-time TEAL template placeholder substitutions (if specified).
-   * @param params The parameters to create the app
-   * @returns The app client and the result of the creation transaction
-   */
-  public async create(params?: AppFactoryCreateParams) {
```

Comment:
Rather than a `create` method that takes bare or abi we split in 2 for consistency with app client. Makes the method less confusing to consume too.